### PR TITLE
LCP-7320

### PR DESCRIPTION
**Pull request by timscottbell**

### CODE REVIEW

**Comment by timscottbell** [comment]
@briangreenwald would you be able to review and merge this to develop, and then to release?

**Comment by briangreenwald** [comment]
Reviewing

File: upgrade-workspace.sh [hunk 1]
```diff
@@ -185,6 +185,11 @@ create_secret() {
   fi
 }
 
+move_custom_service_folders() {
```
Comment (briangreenwald) [inline upgrade-workspace.sh:189]: Does this function need to be called in main()?

**Review by briangreenwald — CHANGES_REQUESTED** [review]

File: upgrade-workspace.sh (hunk 1 above)
Comment (timscottbell) [reply upgrade-workspace.sh:189]: Oops

**Review by briangreenwald — APPROVED** [review]

**Pull request merged**